raaghu-elements/src/rds-input/rds-input.stories.tsx

```
@@ -7,6 +7,22 @@ const meta: Meta = {
   component: RdsInput,
   parameters: {
     layout: 'padded',
+    docs: {
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)